Sources/PT.PM.PythonParseTreeUst/PythonAntlrConverter.cs

```diff
@@ -806,12 +812,35 @@ public Ust VisitName(PythonParser.NameContext context)
 
         public Ust VisitNumber(PythonParser.NumberContext context)
         {
+            if (context.integer() != null)
```

Comment:
Can you please upper-case `integer()` for consistency with `FLOAT_NUMBER()` while you're editing it anyway? :wink:

Reply:
It's not possible, because it comes from generated code by ANTLR. BTW, token names are started with UPPER case letter, parse tree names are started with lower case letter. Thus, `integer` means parse tree node.